lib/gui/index.html

```diff
@@ -57,7 +57,8 @@
         isFinish: state.is('success')
       }">
       <safe-webview
-        src="'https://etcher.io/success-banner/'"></safe-webview>
+        src="'https://etcher.io/'"
+        on-state-change="state.onStateChange"></safe-webview>
```

Comment:
Oh sorry, that's what it does. So `onStateChange` is a higher-order function that passes the state changes to its argument function, and in there we check if the state change is from `'success'` before reloading, visible in the comment code block below.